src/client/containers/collection/index.jsx

```diff
@@ -1,18 +1,35 @@
 import React, { Component } from 'react';
 import './index.css';
+import _ from 'lodash';
```

Comment:
can you import just the shuffle function? Importing the whole library is not efficient for the bundle generated.